e2e/lvmtest/lvmtest.go

```diff
@@ -0,0 +1,188 @@
+package lvmtest
+
+import (
+	"errors"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+)
+
+var (
+	brickPrefix = "/d/backends/"
+	lvmPrefix   = "patchy_snap"
+	pathPrefix  []string
+)
+
+func verifyLVM() bool {
+	out, err := exec.Command("/sbin/lvcreate", "--help").Output()
+	if err != nil {
+		return false
+	}
+	thinLV := strings.Contains(string(out), "thin")
+	return thinLV
+}
+
+func createBrickPath(num int) ([]string, error) {
+
+	var brickPath []string
+
+	for i := 1; i <= num; i++ {
+		prefix := brickPrefix + strconv.Itoa(i) + "/" + lvmPrefix
```

Comment:
don't use hardcoded path, use t.Name()

Reply:
I didn't understand this command, you mean we should avoid paths like brickPrefix = "/d/backends/" and use t.Name() instead ?

Reply:
brickPrefix should be replaced with t.Name() and use fmt.Sprintf()

Reply:
This is the same path name used from the gd1 testing. I kept the same path in case if we are using  our .t fails replacing with glustercli in glusterd2, then it would be easier to port. I guess it is fine to use the brick path, as it just mount it on the location and does a RemoveAll at the end

Reply:
all glusterd2 test cases should use the /tmp/gd2_func_test directory for testing if we create any directory for testing it should be under this path only.

you can use below code to form brickPrefix
```
brickPrefix=baseWorkdir+t.Name()
```